ethereum/rpc/namespaces/cosmos/api.go

```diff
@@ -0,0 +1,180 @@
+package cosmos
```

Comment:
nit, I think we should eventually restructure the packages to be `rpc/namespaces/` since we are not only including ethereum namespaces

Reply:
can be done in a separate PR though

Reply:
#400